security-command-center/snippets/src/main/java/vtwo/iam/GetIamPolicies.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vtwo.iam;
+
+// [START securitycenter_get_iam_policies_v2]
```

Comment:
nit: please do not include `main()` into the code snippet. instructions for executing snippets users get by looking at the sample code in Github. There is no need to show them in the documentation.

Reply:
Unfortunately it is. However, considering the cognitive load on a documentation reader who is trying to see how the code that _calls the library method_ looks like, including into the snippet additional methods is redundant.
The new widget allows users to automate code execution and see results as well as the full code sample with the main() method. Also they can browse the full code samples by opening the code sample location in Guthub with one-click.
To learn more about the use of region tags please look at go/code-snippets-101

Reply:
done